test/unit/test_devices/test_aireos_device.py

```diff
@@ -75,14 +75,6 @@ def test_send_command_timing(aireos_send_command_timing):
     device.native.send_command_timing.assert_called_with(command)
 
 
-def test_send_command_expect(aireos_send_command_expect):
```

Comment:
I think we need to keep these tests, but change the assertion to `device.native.send_command.assert_called_with($args)`. This should be for all of them.

Reply:
I have added back the test and changed the assertions.